app/experimenter/base/context_processors.py

```diff
@@ -11,4 +11,5 @@ def features(request):
     return {
         "FEATURE_MESSAGE_TYPE": settings.FEATURE_MESSAGE_TYPE,
         "FEATURE_ANALYSIS": settings.FEATURE_ANALYSIS,
+        "SENTRY_DSN": settings.SENTRY_DSN,
```

Comment:
I don't really understand how this ropes into anything. Is it used elsewhere to turn on Sentry?

Reply:
Will be consumed here:

https://github.com/mozilla/experimenter/issues/3429